PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+Please try to adhere to the guidelines below as much as possible when submitting your pull request.
+
+- Please verify that your code satisfies the
+[code/documentation quality guidelines](http://scikit-learn.org/stable/developers/contributing.html#coding-guidelines).
+- Please prefix the title of your pull request with `[MRG]` if the
+contribution is complete and should be subjected to a detailed review.
+
+Otherwise:
```

Comment:
Actually I would remove "Otherwise" as mentioned by somebody else previously. Sorry I didn't read that properly initially.